compiler/test-resources/scripting/showArgs.sc

```diff
@@ -1,4 +1,4 @@
-#!/usr/bin/env scala
+#!/usr/bin/env bin/scala
 
 // precise output format expected by BashScriptsTests.scala
 def main(args: Array[String]): Unit =
```

Comment:
Why not here ?
```suggestion
// def main(args: Array[String]): Unit = // MIGRATION: Scala CLI expects `*.sc` files to be straight-line code
```

Reply:
Oh, but if we comment this then it will be the following test compiler/test-resources/scripting/showArgsNu.sc, what is the purpose of this test then ?

Reply:
all files in /scripting are ran by CoursierScalaTests, which are run by the old ScriptingDriver so require a main method

Reply:
I guess we may want to rethink those coursier tests once it can download the new launcher